woocommerce/woocommerce-gutenberg-products-block
# Add: default template for Collection Filters block

## What

This PR add a simple template for Collection Filters block to add all filter blocks with heading as the default template when merchant insert the block to the page.

See #11460

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add the Collections Filiters block to the editor.
2. See price filter, stock filter, attribute filter blocks are added automatically.


## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
